backend/fms_core/viewsets/_utils.py

```diff
@@ -31,13 +31,13 @@ def _list_keys(d: Dict[str, Any]) -> Dict[str, Any]:
 
 
 class FZY(Func):
-    template = "%(function)s('%(search_term)s', %(expressions)s::cstring)"
+    template = "%(function)s(%(expressions)s::cstring)"
     function = "fzy"
 
     def __init__(self, expression, search_term, **extras):
```

Comment:
rename expression to search_field and invert search_field and search_term (don't forget to flip the FZY call too).